src/felles-komponenter/BrukeravhengigTekst.tsx

```diff
@@ -1,23 +1,34 @@
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
 import { useSelector } from 'react-redux';
 
 import { selectErBruker } from '../moduler/identitet/identitet-selector';
 
 interface Props {
-    id: string;
+    lagtInnAv: string;
```

Comment:
Lurer på om denne bare kan være `veileder`eller `bruker` isåfall kunne det vært en type